src/asciidoc/docs/develop/useGit.adoc

```diff
@@ -72,6 +72,8 @@ To clone your Koji repository locally, you need these strings:
 
 You should know your username by heart, and the access token is a 32-character, randomly-generated hexadecimal number, which you need to note in a safe place.
 
+If the project has a backend, you also need the values of the *`*KOJI_PROJECT_ID` and `KOJI_PROJECT_TOKEN` environment variables.
```

Comment:
Add as the third bullet and simplify, e.g.
* If the project has a backend, the values of the `KOJI_PROJECT_ID`...

Clean up the errant asterisks before KOJI_PROJECT_ID.

Reply:
I'm not sure how you mean to simplify that line. I just removed "also".

Reply:
I meant to use the same text I did, for parallel phrasing of the bullets. This change works as is, though.